<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLayoutMarkerWidgetBase</class>
 <widget class="QWidget" name="QgsLayoutMarkerWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 2px; font-weight: bold; background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Marker</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>306</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Main Properties</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QgsSymbolButton" name="mShapeStyleButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Change…</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Symbol</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mRotationGroupBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Marker Rotation</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="mNorthTypeComboBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="mRotationFromMapCheckBox">
            <property name="text">
             <string>Sync with map</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QgsDoubleSpinBox" name="mNorthOffsetSpinBox">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string> °</string>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>North alignment</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsLayoutItemComboBox" name="mMapComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header location="global">qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsLayoutItemComboBox</class>
   <extends>QComboBox</extends>
   <header>qgslayoutitemcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>mShapeStyleButton</tabstop>
  <tabstop>mRotationGroupBox</tabstop>
  <tabstop>mRotationFromMapCheckBox</tabstop>
  <tabstop>mMapComboBox</tabstop>
  <tabstop>mNorthTypeComboBox</tabstop>
  <tabstop>mNorthOffsetSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
